tests/FSharp.Data.DesignTime.Tests/SignatureTests.fs

```diff
@@ -39,37 +40,88 @@ let portableRuntimeAssembly profile = sourceDirectory ++ ".." ++ ".." ++ "bin" +
 
 let getRuntimeRefs = TypeProviderInstantiation.GetRuntimeAssemblyRefs
 
+[<AutoOpen>]
+module Utils = 
+    let isNull x = match x with null -> true | _ -> false
+
+
+/// Simulate a real host of TypeProviderConfig
+type internal DllInfo(path: string) =
+    member x.FileName = path
+
+/// Simulate a real host of TypeProviderConfig
+type internal TcImports(bas: TcImports option, dllInfos: DllInfo list) =
+    member x.Base = bas
+    member x.DllInfos = dllInfos
+
+
+type internal Testing() =
+
+    /// Simulates a real instance of TypeProviderConfig
+    static member MakeSimulatedTypeProviderConfig (resolutionFolder: string, runtimeAssembly: string, runtimeAssemblyRefs: string list) =
```

Comment:
Can you move this to TypeProviderInstantiation.fs so it also works on Test.fsx?

Reply:
@ovatsus I removed that code, thanks